web/js/stats/stats.js

```diff
@@ -0,0 +1,348 @@
+/**
+ * App statistics module.
+ *
+ * Events:
+ * <- STATS_TOGGLE
+ * <- HELP_OVERLAY_TOGGLED
+ *
+ * @version 1
+ */
+const stats = (() => {
+    const modules = [];
+    let tempHide = false;
+
+    // internal rendering stuff
+    const drawFps = 32;
```

Comment:
I think drawFps 10 is enough. Canva Draw is quite heavy I think

Reply:
Not at all. I dropped it from 60 (or your monitor refresh rate) just because it's too distracting.

Reply:
I'll add this value into the settings module which I've started to write.